clients/payment/src/locales/de.json

```diff
@@ -3,7 +3,7 @@
   "common.cancel": "Abbrechen",
   "error.pageNotFound": "Seite nicht gefunden",
   "form.invalidField": "Ungültiges Feld",
-  "paymentLink.addressLine1": "Adresszeile",
+  "paymentLink.addressLine1": "Adresse",
```

Comment:
Is this german or french?

Reply:
It's German, I removed initially "adress line" for "address" so that's the same change for German